dashboard/test/models/pd/enrollment_test.rb

```diff
@@ -89,6 +89,27 @@ class Pd::EnrollmentTest < ActiveSupport::TestCase
     assert_equal [enrollment_in_district], Pd::Enrollment.for_school_district(school_info.school_district)
   end
 
+  test 'pre_workshop_survey_url' do
+    csp_summer_workshop = build :csp_summer_workshop
+    csp_summer_workshop_enrollment = build :pd_enrollment, workshop: csp_summer_workshop
+
+    csp_academic_year_workshop = build :csp_academic_year_workshop
+    csp_academic_year_workshop_enrollment = build :pd_enrollment, workshop: csp_academic_year_workshop
+
+    csf_deep_dive_workshop = build :csf_deep_dive_workshop
+    csf_201_workshop_enrollment = build :pd_enrollment, workshop: csf_deep_dive_workshop
+
+    csf_intro_workshop = build :csf_intro_workshop
+    csf_intro_workshop_enrollment = build :pd_enrollment, workshop: csf_intro_workshop
+
+    assert_equal "/pd/workshop_survey/day/0?enrollmentCode=#{csp_summer_workshop_enrollment.code}",
```

Comment:
This is a bit of a hack for the fact that `url_for` was generating HTTP URLs, and our helper `CDO.studio_url(relative_path` generated HTTPS.